Assets/MixedRealityToolkit.SDK/Features/Utilities/PointerUtils.cs

```diff
@@ -144,6 +144,159 @@ public static IEnumerable<IMixedRealityPointer> GetPointers()
                 }
             }
         }
-    }
 
+        /// <summary>
+        /// Gets the physics ray based on the interaction enabled pointer assosiciated with the head gaze.
```

Comment:
I actually never thought of rays as "physics ray". "Physics ray" to me implies that it's somehow part of Unity's physics system, meaning it's like a RigidBody or something fancy like that. A ray is just a position and a direction. Also, I would remove "interaction enabled pointer" since that confused even me, imagine what somebody new would think :-)

How about:

Gets the position and direction of the head.

The goal here is to make this an easy / understandable / approachable thing for people not familiar with MRTK. So, the way I'd think about it is "would somebody new to MRTK understand what this method does".

Reply:
fixed